### ISSUE TITLE
Fresh install, wrong zoom ratio, Nyxt won't start

### ISSUE THREAD

**Issue by ixzh** [atlas-engineer/nyxt#1472]
**Describe the bug**
fresh install, set default room ratio "125%", indicated unbound error. restart Nyxt, window won't appear.

**Precise recipe to reproduce the issue**
as above

**Information**
- OS name+version: Arch
- Graphics card and driver: Intel Haswell-ULT Integrated Graphics
- Desktop environment / Window manager name+version: xmonad
- How you installed Nyxt (Guix pack, package manager, build from source): AUR
- Information from command copy-system-information:

  If you can't run copy-system-information, try `nyxt --system-information` from
  a shell.  It this still does not work, please provide the following:
  - Nyxt version (from =M-x nyxt-version= or =nyxt --version=):
  - Lisp implementation/version (if built from source):
  - Kernel name+version:
  - WebKitGTK+ | QtWebEngine version:

**Output when started from a shell**

**Comment by ixzh** [comment]
running from commandline gave some error info because of auto-config.lisp:

![image](https://user-images.githubusercontent.com/14981695/120910624-eeee3900-c6b2-11eb-944f-24dcbfddd455.png)
It registered all the wrong value setting....seems every button click,even if repeatedly, are registered repeatedly.

**Comment by aartaka** [comment]
Registering every click is intentional.

It's strange that it got appended to configuration. It shouldn't. I'll push the type-strengthening change disallowing this value in an hour.

**Comment by aartaka** [comment]
Okay, I've added more types to buffer slots (in bf2167c95d47a5d049c5936a43e60e79df7ee70b) and made `configure-slot` stricter about types (in ac9722fc5998b7a58ba860dec067c8d8b22ef997). 

In plain English: Nyxt should now warn you when you try to put something it does not expects, into a slot. But to have this, you need to wait for a release or build from master branch :(

**Comment by Ambrevar** [comment]
Also Nyxt should start anyways when define-configuration is used over wrong
values.

See https://github.com/atlas-engineer/nyxt/issues/1498.
